<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerManage</class>
 <widget class="QWidget" name="SellerManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>771</width>
     <height>391</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sex</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Branch ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Basic Salary</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Percentage</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Seller</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Seller</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>SellerManage</receiver>
   <slot>onAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>SellerManage</receiver>
   <slot>onDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>SellerManage</receiver>
   <slot>onApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>SellerManage</receiver>
   <slot>onResetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddClicked()</slot>
  <slot>onDeleteClicked()</slot>
  <slot>onResetClicked()</slot>
  <slot>onApplyClicked()</slot>
 </slots>
</ui>
